### PR TITLE
Updates snapcraft.io Vanilla to 1.7.1

### PR DESCRIPTION
**Pull request by bartaz**
Updates Vanilla to 1.7.1

We need to QA it and fix any layout issues found.

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ $color-navigation-background: #252525;
 
 @include vf-base;
 
+@include vf-u-hide; // required by vf-p-icons
```
Comment (bartaz) [inline static/sass/styles.scss]: Needed because of https://github.com/vanilla-framework/vanilla-framework/issues/1821

Comment (Caleb-Ellis) [reply static/sass/styles.scss]: @bartaz Hotfixed as part of 1.7.1 release

**Comment by bartaz** [comment]
@lyubomir-popov Thanks for updates. Yes - I realised that our `p-heading--` hacks on home page are making everything unnecessarily bold now.

I also noticed some unnecessary `u-no-margin`, but that's also an easy fix.

Also - are there any more specific docs about new spacing variables?

**Comment by lyubomir-popov** [comment]
@Lukewh @bartaz what is the plan with this branch? Is work planned in any of the next few iterations?

**Comment by bartaz** [comment]
### Snap details (http://snapcraft.io-pr-672.run.demo.haus/phpstorm)

- [ ] bigger snap name
- [ ] too much spacing between snap name and publisher
- [ ] Install/Versions buttons seem misaligned with snap name
- [ ] No spacing under screenshots
- [ ] Next/prev screenshot button position changed (?)
- [ ] In lightbox screenshot is not centered vertically
- [ ] Messed up dotted borders in the table

screen shot 2018-06-14 at 12 49 33

- [ ] double border under tabs in channel map overlay
- [ ] messed up spacing in install tab
- [ ] messed up spacing in channel map tab

screen shot 2018-06-14 at 12 52 19
screen shot 2018-06-14 at 12 51 59

**Comment by lyubomir-popov** [comment]
Hey @bartaz how would you prefer comments on the above? In the thread here, or maybe set up a meeting to talk about them?

**Comment by bartaz** [comment]
@lyubomir-popov I'm just testing right now, possibly we will create separate issues for them. We plan to work on it next iteration, so no rush on that. But I guess it would be good to have a call at some point for discussing the issues and getting your tips on how to fix them (and what changed to Vanilla spacing).

**Comment by bartaz** [comment]
### Account snaps (http://snapcraft.io-pr-672.run.demo.haus/account/snaps)

- [ ] 'beta' banner paragraph has additional bottom spacing
- [ ] 'My snaps' and docs link on the right are misaligned
- [ ] Name column in snaps table is very narrow 
- [ ] My registered names (top line) is close to the table and looks like part of it

screen shot 2018-06-14 at 13 12 40

**Comment by lyubomir-popov** [comment]
What is the url for the current version of this pr? I tried a couple of the links above but they seem to have changed

**Pull request merged**